OpenTrons/opentrons-api
# fix(app, odd): run summery error list bug fixes

# Overview

Closes [RQA-2923](https://opentrons.atlassian.net/browse/RQA-2923).
show warnings/errors depending on the run status.
modal header.
show error details on ODD if run completed with errors.

## Test Plan and Hands on Testing

- run a protocol with ER errors on ODD and desktop app.
- make sure that when the run is complete you are able to see the list of errors.
- warnings shown when run is successfully completed, errors if canceled or failed.

## Changelog

- inject run status to `RunFailedModal`.
- changed modal title
- changed text to show warnings/errors depending on the run status. 
- ODD - show view error details if there is an error list.

## Risk assessment

low. bug fixes.


[RQA-2923]: https://opentrons.atlassian.net/browse/RQA-2923?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
